site-icon-as-image/snippet.html

```diff
@@ -0,0 +1,18 @@
+<div id="siteIcon"></div>
```

Comment:
Use of ID's in FrogOS is prohibited unless the ID is truly unique (contains a timestamp or uses `Frog.Utilities.generateSimpleId()`)